src/providers/consensus/types.py

```diff
@@ -168,11 +151,43 @@ class SlotAttestationCommittee(FromResponse):
     validators: list[str]
 
 
+@dataclass
+class PendingDeposit(Nested):
+    pubkey: str
+    withdrawal_credentials: str
+    amount: Gwei
+    signature: str
+    slot: SlotNumber
+
+
+@dataclass
+class PendingPartialWithdrawal(Nested):
+    validator_index: str
+    amount: Gwei
+    withdrawable_epoch: EpochNumber
+
+
 @dataclass
 class BeaconStateView(Nested, FromResponse):
     """A view to BeaconState with only the required keys presented"""
```

Comment:
add link to spec
https://github.com/ethereum/consensus-specs/blob/dev/specs/electra/beacon-chain.md#beaconstate